process/smartContract/process.go

```diff
@@ -1172,18 +1164,10 @@ func (sc *scProcessor) DeploySmartContract(tx data.TransactionHandler, acntSnd s
 		return 0, err
 	}
 
-	if !sc.flagDeploy.IsSet() {
-		err = sc.updateDeveloperRewardsV1(tx, vmOutput, 0)
-		if err != nil {
-			log.Debug("updateDeveloper rewards error", "error", err.Error())
-			return 0, err
-		}
-	} else {
-		err = sc.updateDeveloperRewardsV2(tx, vmOutput, 0)
-		if err != nil {
-			log.Debug("updateDeveloper rewards error", "error", err.Error())
-			return 0, err
-		}
+	err = sc.updateDeveloperRewardsProxy(tx, vmOutput, 0)
+	if err != nil {
+		log.Debug("updateDeveloper rewards error", "error", err.Error())
```

Comment:
To avoid printing twice the word "error" you can replace this line with:
log.Error("updateDeveloperRewardsProxy", "error", err.Error())

Reply:
done